### ISSUE TITLE
TSBToolSupreme 1.3.0.1 - Player Sim attributes for TSB3 snes players won’t get saved.

### ISSUE THREAD

**Issue by Denshadego** [BAD-AL/tsbtools#2]
Thanks for maintaining this awesome tool, the issue I previously reported is indeed fixed. However, when doing the auto updating of player SIM attributes for testing purposes, applying the changes to the rom and then reloading the rom, shows the player Sim stats as they were initially with no changes applied. This was using a TSB3 rom (The same one uploaded on the previous issue)

**Comment by BAD-AL** [comment]
you're right.

You don't mind being the main tester for TSB3, do you?

**Comment by Denshadego** [comment]
No problem at all. While I don't have that much free time, I'm working on updating the game's rosters and this tool shows promise on that regard (Most tools center on TSB1 NES/SNES and the few for TSB3 are much less versatile than those). I'll report on any bugs I may encounter while at it.

**Comment by BAD-AL** [comment]
Looking at the code (which was written a couple years ago now), it appears that I had the sim value saving implemented at some point and then 'commented it out'.
I'm not sure why though. It could be that the SIM stuff was just not really confirmed.
Here's a new exe to test
[TSBToolSupreme_TEST_TSB2-3_simData.zip](https://github.com/BAD-AL/tsbtools/files/5978319/TSBToolSupreme_TEST_TSB2-3_simData.zip)

**Comment by Denshadego** [comment]
Yeah, there's no tool that handles TSB3 sim values beyond manually hex editing, so when it didn't work I assumed it may be a leftover option from TSB1. Although, after giving the new exe attached a go, applying changes to a rom and reloading, the new values seem to stay. Did a couple of sim/skip matches to test and nothing out of the ordinary yet, so it seems good for now.

**Comment by BAD-AL** [comment]
Yes, the 'saving' works in this one. But do the sim values do what we think they should do?

**Comment by Denshadego** [comment]
Yeah, you're right, they do save the values to show up in the editor, but the teams behavior in simmed games is the same as before editing.
I changed all Bears player attributes to the minimum allowed, and as they were on week one, did the game on skip a bunch of times and the results are similar to a rom with no update, generally losing 14-41 , but surprisingly a couple of times won, which should be impossible. In contrast, when matching the same teams COM vs COM, the Bears lost 0-81 which was expected.
Could the editor be pointing to the wrong location/s on the rom?

**Comment by BAD-AL** [comment]
The sim values as the program knows were discussed in this thread:
https://tecmobowl.org/forums/topic/53028-tecmo-super-bowl-ii-hackingresource-documentation/
Perhaps we've got the wrong stuff?
